riff-raff/app/deployment/Deployments.scala

```diff
@@ -60,10 +63,11 @@ class Deployments(deploymentEngine: DeploymentEngine,
   def getStopFlag(uuid: UUID): Boolean = deploymentEngine.getDeployStopFlag(uuid)
 
   def create(params: DeployParameters): Record = {
-    log.info(s"Creating deploy record for $params")
     val uuid = java.util.UUID.randomUUID()
     val hostNameMetadata = Map(Record.RIFFRAFF_HOSTNAME -> java.net.InetAddress.getLocalHost.getHostName)
     val record = deployRecordFor(uuid, params) ++ hostNameMetadata
+    val mc = MarkerContext(appendEntries(record.metaData.asJava))
+    log.info(s"Creating deploy record for $params")(mc)
```

Comment:
This is the meaningful change in the branch. The rest is my IDE being helpful.

Reply:
👍 for not creating an extra log line. 

Am I right in thinking that this will just splat the metadata map keys into elk? Do we want to do that (given the chat recently about ES common schema etc?

Reply:
Is `labels` the right place for this? https://www.elastic.co/guide/en/ecs/current/ecs-base.html

Reply:
I don't know enough about ECS to be confident in this response...

I think ECS is a sort of all or nothing deal. That is, we'd need the entire app/log producers in the index to adhere to the schema to see the benefits of it. With this in mind, I'd be tempted to push that work to a later time as it's quite big.